src/pom.xml

```diff
@@ -72,27 +72,6 @@
     <url>http://download.osgeo.org/webdav/geotools/</url>
     <!-- release repository used by geotools (and third-party dependencies) -->
   </repository>
-   
-  <repository>
-    <id>maven-restlet</id>
-    <name>Restlet Maven Repository</name>
-    <url>http://maven.restlet.org</url>
```

Comment:
This repo is working for me, but it is cached (https://repo.boundlessgeo.com/artifactory/webapp/#/artifacts/browse/tree/General/restlet-cache) and republished as part of repo.boundlessgeo.com/main